### ISSUE TITLE
Links look like regular text in High Contrast White Theme

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1035]
**Describe the bug**
In the Fiori 3 'high contrast white' theme - Links look like regular text.

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/link.html
2. It is not possible to visually distinguish links from regular text because of loss of color.

**Expected behavior**
Users should be able to visually distinguish between a link and normal text in a Fiori 3 high-contrast white theme. Either use a different color (blue) or use some type of text-decoration. Using a different color is recommended.

**Screenshots**
Links in the normal theme have a blue color
![image](https://user-images.githubusercontent.com/43764832/81754583-69766600-946b-11ea-8a84-b85ad2349c24.png)

Links in the Fiori 3 HCW theme have white color which is the same as normal text.
![image](https://user-images.githubusercontent.com/43764832/81859710-828a2000-951a-11ea-8971-70988ee5ec38.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by droshev** [comment]
the issue has been addressed as part of the delta theming
